server/events/vcs/bitbucketcloud/models.go

```diff
@@ -51,6 +51,7 @@ type PullRequest struct {
 	Participants []Participant `json:"participants,omitempty" validate:"required"`
 	Links        *Links        `json:"links,omitempty" validate:"required"`
 	State        *string       `json:"state,omitempty" validate:"required"`
+	Author       *Author       `jsonN:"author,omitempty" validate:"required"`
```

Comment:
@lkysow This `jasoN` looks strange. However, I am not sure if this will make a problem here.